website/docs/r/fallback_domain.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "cloudflare"
+page_title: "Cloudflare: cloudflare_fallback_domain"
+sidebar_current: "docs-cloudflare-resource-fallback-domain"
```

Comment:
don't forget to add a link this to document in https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/website/cloudflare.erb so it appears in the sidebar.